modules/aerodyn/src/AeroDyn_Inflow.f90

```diff
@@ -405,125 +451,6 @@ subroutine concatOutputHeaders(WriteOutputHdr0, WriteOutputUnt0, WriteOutputHdr,
-!----------------------------------------------------------------------------------------------------------------------------------
-!> Set inputs for inflow wind
-subroutine ADI_Set_IW_Inputs(p_AD, u_AD, o_AD, u_IfW, hubHeightFirst, errStat, errMsg)
-   type(ADI_ParameterType),      intent(in   ) :: p_AD          ! Parameters
-   type(AD_InputType),           intent(in   ) :: u_AD          ! AeroDyn data 
-   type(AD_OtherStateType),      intent(in   ) :: o_AD          ! AeroDyn data 
-   type(InflowWind_InputType),   intent(inout) :: u_IfW         ! InflowWind data 
-   logical,                      intent(in   ) :: hubHeightFirst ! Hub Height velocity is packed at beginning
-   integer(IntKi)              , intent(  out) :: errStat       ! Status of error message
-   character(*)                , intent(  out) :: errMsg        ! Error message if errStat /= ErrID_None
-   integer :: node, iWT
-   errStat = ErrID_None
-   errMsg  = ''
-   node=0
-
-   if (hubHeightFirst) then
-      ! Hub Height point for each turbine
-      do iWT=1,size(u_AD%rotors)
-         node = node + 1
-         u_IfW%PositionXYZ(:,node) = u_AD%rotors(iWT)%hubMotion%Position(:,1) + u_AD%rotors(iWT)%hubMotion%TranslationDisp(:,1)
-      enddo
-   endif
-   call AD_SetExternalWindPositions(u_AD, o_AD, u_IfW%PositionXYZ, node, errStat, errMsg)
-   if ( p_AD%MHK == 1 .or. p_AD%MHK == 2 ) then
-      u_IfW%PositionXYZ(3,:) = u_IfW%PositionXYZ(3,:) + p_AD%WtrDpth
-   endif 
-end subroutine ADI_Set_IW_Inputs
-!----------------------------------------------------------------------------------------------------------------------------------
-!----------------------------------------------------------------------------------------------------------------------------------
-!> Calculate Wind at desired points
-!! NOTE: order is important and should match AD_NumWindPoints
-!! Similar to FAST_Solver, IfW_InputSolve
-subroutine ADI_CalcOutput_IW(t, u_IfW, IW, errStat, errMsg)
-   real(DbKi),                   intent(in   ) :: t             ! Time of evaluation
-   type(InflowWind_InputType),   intent(inout) :: u_IfW         ! InflowWind data 
-   type(ADI_InflowWindData),     intent(inout) :: IW            ! InflowWind data 
-   integer(IntKi)              , intent(  out) :: errStat       ! Status of error message
-   character(*)                , intent(  out) :: errMsg        ! Error message if errStat /= ErrID_None
-   integer              :: j
-   real(ReKi)           :: z
-   integer(IntKi)       :: errStat2      ! Status of error message
-   character(errMsgLen) :: errMsg2       ! Error message if errStat /= ErrID_None
-   errStat = ErrID_None
-   errMsg  = ''
-   if (IW%CompInflow==1) then
-      call InflowWind_CalcOutput(t, u_IfW, IW%p, IW%x, IW%xd, IW%z, IW%OtherSt, IW%y, IW%m, errStat2, errMsg2)
-      call SetErrStat(errStat2, errMsg2, errStat, errMsg, 'ADI_CalcOutput_IW') 
-   else
-      !$OMP PARALLEL DEFAULT(SHARED)
-      !$OMP DO PRIVATE(j,z) schedule(runtime)
```

Comment:
Here the OMP parallelization is likely lost.

Reply:
I agree that the calculation won't be in parallel in OMP. I'd like to explore adding OpenMP within the FlowField, which is inside InflowWind.